coalib/__init__.py

```diff
@@ -6,6 +6,10 @@
 
 
 def get_version():
+    '''The coalib package is a collection of various subpackages
```

Comment:
Can you use double quotes? Also a newline after the triple quotes and before the ending ones is what we do usually